src/server/JasmineGraphInstanceService.cpp

```diff
@@ -80,4301 +144,5243 @@ void *instanceservicesession(void *dummyPt) {
+                instance_logger.log("Sent : " + JasmineGraphInstanceProtocol::FILE_ACK, "info");
+            }
+
+            Utils::unzipDirectory(fullFilePath);
+            size_t lastindex = fileName.find_last_of(".");
+            string pre_rawname = fileName.substr(0, lastindex);
+            size_t next_lastindex = pre_rawname.find_last_of(".");
+            string rawname = fileName.substr(0, next_lastindex);
+            fullFilePath =
+                Utils::getJasmineGraphProperty("org.jasminegraph.server.instance.trainedmodelfolder") + "/" + rawname;
 
+            while (!Utils::fileExists(fullFilePath)) {
+                bzero(data, INSTANCE_DATA_LENGTH + 1);
+                read(sockfd, data, INSTANCE_DATA_LENGTH);
+                string response = (data);
+                response = Utils::trim_copy(response, " \f\n\r\t\v");
+                if (response.compare(JasmineGraphInstanceProtocol::BATCH_UPLOAD_CHK) == 0) {
+                    instance_logger.log("Received : " + JasmineGraphInstanceProtocol::BATCH_UPLOAD_CHK, "info");
+                    write(sockfd, JasmineGraphInstanceProtocol::BATCH_UPLOAD_WAIT.c_str(),
+                          JasmineGraphInstanceProtocol::BATCH_UPLOAD_WAIT.size());
+                    instance_logger.log("Sent : " + JasmineGraphInstanceProtocol::BATCH_UPLOAD_WAIT, "info");
+                }
+            }
             bzero(data, INSTANCE_DATA_LENGTH + 1);
-            read(connFd, data, INSTANCE_DATA_LENGTH);
-            string alphaValue = (data);
-            alphaValue = Utils::trim_copy(alphaValue, " \f\n\r\t\v");
-            instance_logger.log("Received alpha: " + alphaValue, "info");
+            read(sockfd, data, INSTANCE_DATA_LENGTH);
+            response = (data);
+            if (response.compare(JasmineGraphInstanceProtocol::BATCH_UPLOAD_CHK) == 0) {
+                instance_logger.log("Received : " + JasmineGraphInstanceProtocol::BATCH_UPLOAD_CHK, "info");
+                write(sockfd, JasmineGraphInstanceProtocol::BATCH_UPLOAD_ACK.c_str(),
+                      JasmineGraphInstanceProtocol::BATCH_UPLOAD_ACK.size());
+                instance_logger.log("Sent : " + JasmineGraphInstanceProtocol::BATCH_UPLOAD_ACK, "info");
+            }
+        }
+    } else {
+        instance_logger.log("There was an error in the model collection process and the response is :: " + response,
+                            "error");
+    }
 
-            double alpha = std::stod(alphaValue);
+    close(sockfd);
+    return 0;
+}
 
-            write(connFd, JasmineGraphInstanceProtocol::OK.c_str(), JasmineGraphInstanceProtocol::OK.size());
-            instance_logger.log("Sent : " + JasmineGraphInstanceProtocol::OK, "info");
+void JasmineGraphInstanceService::createPartitionFiles(std::string graphID, std::string partitionID,
+                                                       std::string fileType) {
+    instance_logger.log(fileType, "info");
```

Comment:
This log message has no meaning. Please improve it.

Reply:
Replaced with `instance_logger.info("createPartitionFiles called with fileType=" + fileType);` in next PR

Reply:
Ok